models/modelo.ts

```diff
@@ -0,0 +1,27 @@
+import { Tiempo } from "./tiempo.ts";
+import { Operacion } from "./operacion.ts";
+
+//Conjunto de operaciones necesarias para la fabricación de un modelo concreto
+export class Modelo{
+    
+    //Concepto del modelo
+    concepto: string;
+
+    //Conjunto de operaciones
+    operaciones: Array<Operacion>;
+
+    //Duracion estimada para la fabricación si solo hay un trabajador
+    duracion_estimada_lineal: Tiempo;
+
+    constructor(concepto:string, operaciones: Array<Operacion>){
+        this.concepto = concepto;
+        this.operaciones = operaciones;
+        this.duracion_estimada_lineal = new Tiempo();
+
+        //Sumamos la duracion estimada de todas las operaciones
+        operaciones.forEach(operacion => {
+            this.duracion_estimada_lineal.sumar(operacion.duracion_estimada)
```

Comment:
```suggestion
    duracion_estimada_lineal: number;

    constructor(concepto:string, operaciones: Array<Operacion>){
        this.concepto = concepto;
        this.operaciones = operaciones;
        this.duracion_estimada_lineal = new Tiempo();

        //Sumamos la duracion estimada de todas las operaciones
        operaciones.forEach(operacion => {
            this.duracion_estimada_lineal += operacion.duracion_estimada;
```
@vdeq79 tras revisar detenidamente la clase Tiempo, ya que realmente no es necesario acceder durante la lógica de negocio a cada propiedad de la clase tiempo, sería más interesante que fuera simplemente un campo numérico tipo timestamp incluyendo la totalidad de segundos que se tardaría. De este modo, no sería necesario implementar todas las validaciones derivadas de los cálculos que se realizan en dicha clase y también se simplificaría la serialización de los modelos ya sea para implementar una interfaz REST, almacenar la información en base de datos, etc.